### DISCUSSION TITLE
How to increase the execution timeout for both PHP and nginx

### DISCUSSION THREAD

**Discussion by GreenLondon** [EasyEngine/easyengine#1715] [Support Requests]
We have custom php project where the user can generate a big report.

We optimised the code all we could based on the requirements but now we have to increase the PHP and nginx time out.

We managed to increase the PHP one but nginx cuts the query at some point. 

How to increase the timeout for nginx just for that site?

**Comment by GreenLondon** [comment]
A reply will be much appreciated

**Comment by marcoandrei** [comment]
@GreenLondon 

Have you solve your problem? I have the same problem. Thanks!
